Backend/app/services/alert_monitoring_service.py

```
@@ -0,0 +1,220 @@
+            db.close()
+
+    async def check_campaign_now(self, campaign_id: str):
+        """Manually trigger alert checking for a specific campaign."""
+        try:
+            db = next(get_db())
+            
+            # Get active alerts for this campaign
+            alerts = db.query(AlertConfig).filter(
+                and_(
+                    AlertConfig.campaign_id == campaign_id,
+                    AlertConfig.is_active == True
```

Ruff (0.13.1)

Avoid equality comparisons to `True`; use `AlertConfig.is_active:` for truth checks

Replace with `AlertConfig.is_active`

(E712)